src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,31 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import java.util.Optional;
+import mate.academy.dao.BookDao;
+import mate.academy.dao.Injector;
+import mate.academy.model.Book;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.academy.dao");
+
     public static void main(String[] args) {
+        BookDao bookDao = (BookDao) injector.getInstance(BookDao.class);
+        Book newBook = new Book(null, "Effective Java", new BigDecimal("39.99"));
+        bookDao.create(newBook);
+
+        // Find a book by ID
+        Optional<Book> foundBook = bookDao.findById(newBook.getId());
```

Comment:
Ensure that `newBook.getId()` returns a valid ID. If the ID is generated by the database upon creation, you need to retrieve it after the `create` method call.